diff --git a/data/meson.build b/data/meson.build
index 33f73ca..8974a63 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -17,7 +17,6 @@ install_data(
 
 # Desktop file
 desktop_file = i18n.merge_file(
-  'desktop-file',
   input: 'org.gnome.TwentyFortyEight.desktop.in',
   output: 'org.gnome.TwentyFortyEight.desktop',
   install: true,
@@ -37,7 +36,6 @@ endif
 
 # AppData file
 appdata_file = i18n.merge_file(
-  'appdata-file',
   input: 'org.gnome.TwentyFortyEight.appdata.xml.in',
   output: 'org.gnome.TwentyFortyEight.appdata.xml',
   install: true,
diff --git a/data/org.gnome.TwentyFortyEight.appdata.xml.in b/data/org.gnome.TwentyFortyEight.appdata.xml.in
index 007fc99..86c9932 100644
--- a/data/org.gnome.TwentyFortyEight.appdata.xml.in
+++ b/data/org.gnome.TwentyFortyEight.appdata.xml.in
@@ -42,8 +42,9 @@
   <launchable type="desktop-id">org.gnome.TwentyFortyEight.desktop</launchable>
   <url type="homepage">https://wiki.gnome.org/Apps/2048</url>
   <url type="bugtracker">https://gitlab.gnome.org/GNOME/gnome-2048/issues</url>
-  <url type="donation">https://www.gnome.org/friends/</url>
-  <url type="translate">https://wiki.gnome.org/TranslationProject</url>
+  <url type="donation">https://www.gnome.org/donate/</url>
+  <url type="vcs-browser">https://gitlab.gnome.org/GNOME/gnome-2048</url>
+  <url type="translate">https://l10n.gnome.org/module/gnome-2048/</url>
   <update_contact>arnaud.bonatti_at_gmail.com</update_contact>
   <project_group>GNOME</project_group>
   <!-- this is a translatable version of project_group-->
@@ -56,44 +57,15 @@
   </kudos>
   <translation type="gettext">gnome-2048</translation>
 
-  <content_rating type="oars-1.1">
-    <content_attribute id="violence-cartoon">none</content_attribute>
-    <content_attribute id="violence-fantasy">none</content_attribute>
-    <content_attribute id="violence-realistic">none</content_attribute>
-    <content_attribute id="violence-bloodshed">none</content_attribute>
-    <content_attribute id="violence-sexual">none</content_attribute>
-    <content_attribute id="violence-desecration">none</content_attribute>
-    <content_attribute id="violence-slavery">none</content_attribute>
-    <content_attribute id="violence-worship">none</content_attribute>
-    <content_attribute id="drugs-alcohol">none</content_attribute>
-    <content_attribute id="drugs-narcotics">none</content_attribute>
-    <content_attribute id="drugs-tobacco">none</content_attribute>
-    <content_attribute id="sex-nudity">none</content_attribute>
-    <content_attribute id="sex-themes">none</content_attribute>
-    <content_attribute id="sex-homosexuality">none</content_attribute>
-    <content_attribute id="sex-prostitution">none</content_attribute>
-    <content_attribute id="sex-adultery">none</content_attribute>
-    <content_attribute id="sex-appearance">none</content_attribute>
-    <content_attribute id="language-profanity">none</content_attribute>
-    <content_attribute id="language-humor">none</content_attribute>
-    <content_attribute id="language-discrimination">none</content_attribute>
-    <content_attribute id="social-chat">none</content_attribute>
-    <content_attribute id="social-info">none</content_attribute>
-    <content_attribute id="social-audio">none</content_attribute>
-    <content_attribute id="social-location">none</content_attribute>
-    <content_attribute id="social-contacts">none</content_attribute>
-    <content_attribute id="money-purchasing">none</content_attribute>
-    <content_attribute id="money-gambling">none</content_attribute>
-  </content_rating>
+  <content_rating type="oars-1.1" />
 
   ​<releases>
     <!-- should be kept in versioning inverted order -->
     <release version="3.38.2" date="2020-11-22" type="stable" urgency="medium"/>
   ​  <release version="3.37.91" date="2020-08-28" type="development" urgency="low"/>
   </releases>
 
   <custom>
-    <value key="Purism::form_factor">workstation</value>
     <value key="Purism::form_factor">mobile</value>
   </custom>
 </component>
